### PR TITLE
Te starting on UI

### PR DESCRIPTION
**Pull request by Testell**

### CODE REVIEW

**Comment by Testell** [comment]
My checks will fail until the last assignment when I pass all the tests. Please ignore that for now and just view changes

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -1,57 +1,41 @@
...
+
+        <% else %>
+          <li class="nav-item">
+            <%= link_to "Sign in", new_user_session_path, class: "nav-link" %>
+          </li>
+          <li class="nav-item">
+            <%= link_to "Sign up", new_user_registration_path, class: "nav-link" %>
+          </li>
+
+        <% end %>
```
Comment (jessjno) [inline app/views/shared/_navbar.html.erb:38]: I think you need a closing ul tag here

**Review by jessjno — APPROVED** [review]
nice!

File: app/views/shared/_navbar.html.erb (hunk 1 above)
Comment (aldoportillo) [reply app/views/shared/_navbar.html.erb:38]: Isn't it on line 39?

File: config/routes.rb [hunk 2]
```diff
@@ -1,11 +1,10 @@
 Rails.application.routes.draw do
-  resources :likes
-  resources :follow_requests
+  root "photos#index"
```
Comment (aldoportillo) [inline config/routes.rb:2]: good

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -16,8 +16,15 @@
 
   <%= render partial: "shared/navbar"%>
 
-   <div class="container mt-3">
-      <%= render partial: "shared/flash_messages" %>
+   <div class="container">
```
Comment (aldoportillo) [inline app/views/layouts/application.html.erb:19]: good

File: app/views/shared/_navbar.html.erb [hunk 4]
```diff
@@ -1,57 +1,42 @@
...
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <div class="dropdown-menu">
```
Comment (aldoportillo) [inline app/views/shared/_navbar.html.erb:14]: nice

**Review by aldoportillo — APPROVED** [review]

**Review by dsimp — COMMENTED** [review]
Good stuff

**Pull request merged**